portfolio/src/main/webapp/script.js

```diff
@@ -94,6 +94,7 @@ function showCaption(element) {
   .then((login) => {
     const commentsDiv = document.getElementById('comments');
 
+
```

Comment:
(This is the only place that this file is allowing me to add this comment to) 
I would recommend breaking out any of the comments related functions into their own comments.js file. That helps avoid making this file super long and helps with context/readability as you know that comments.js should only contain code related to the comment section.